src/components/VideoCard.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import { Video } from "../types";
+import { ArrowRightCircleFillIcon } from "./ArrowRightCircleFillIcon";
+
+const getDurationInString = (seconds: number) => {
+  if (seconds < 60) {
+    return `${seconds} seconds`;
+  }
+  const minutes = Math.floor(seconds / 60);
+  return `${minutes} minute${minutes > 1 ? "s" : ""}`;
```

Comment:
You're making an assumption that duration is only in minutes. Is that true? For brevity, I would check with UX whether it's better to use the actual numbers `5:55` like Youtube.